spec.md

```diff
@@ -487,6 +490,92 @@ The tags MUST be in lexical order.
 
 When using the `last` query parameter, the `n` parameter is OPTIONAL.
 
+##### Listing Referrers
+
+*Note: this feature was added in distibution-spec 1.1.
+Clients should see [client-implementation.md](client-implementation.md) for more details on implementing this.
```

Comment:
cross referencing two documents, bouncing back and forth, to find out the total of what must be implemented makes it hard for reader/implementor.

Reply:
The current spec felt focused on registry server concerns. Adding details for what clients should do when a server doesn't provide the API felt client specific, and not really part of the API spec. Should these two be merged, or is there a better way to separate the interest?

Reply:
it's currently merged as one api reference.. see references to client(s) required actions..

Reply:
Okay, I can work on moving this into one doc. To verify, you're not concerned this is talking about how to integrate with 1.0 registries in the 1.1 spec?

Reply:
nod, since the changes put requirements on the existing api implementations (client and server) I think we need those requirements to be in one doc.. 

I also think it's fair to include how to integrate with 1.0 registries as a separate section for push.. just have to high light "for clients implementing this version of the spec > 1.0 care MUST be taken to be backwards compatible with 1.0 registries that do not have support for artifact manifests, refers in image manifests, and artifact manifests in image.index.."